integration/telegrambot/telegrambot.go

```diff
@@ -0,0 +1,41 @@
+package telegrambot
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"strconv"
+
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+	"miniflux.app/model"
+)
+
+// PushEntry pushes entry to telegram chat using integration settings provided
+func PushEntry(entry *model.Entry, integration *model.Integration) error {
```

Comment:
It would be better if this function was not coupled with  `*model.Integration`.  You can pass the bot token and chat ID as arguments instead (similar to other integration).